Cross.toml

```diff
@@ -1,14 +1,14 @@
-# Cross.toml
-
 [target.x86_64-unknown-linux-gnu]
+image = "ubuntu:22.04"
```

Comment:
Might as well use 24.04 at this point, that's also what Github's `ubuntu-latest` currently points to